tests/testapps/CTCatalog/CTCatalog/Tests/CTCFramesetterTestViewController.mm

```diff
@@ -82,7 +86,10 @@ - (void)drawRect:(CGRect)rect {
     CGContextAddRect(context, rect);
     CGContextStrokePath(context);
 
+    CGPathRelease(path);
     CGColorSpaceRelease(colorspace);
+
+    CFRetain(framesetter);
```

Comment:
why do the refreshValuesFor... functions release this? that seems like a pretty weird memory management pattern?

---
In reply to: [79930597](https://github.com/Microsoft/WinObjC/pull/1010#discussion_r79930597) [](ancestors = 79930597)